README.md

```diff
@@ -3,29 +3,30 @@
 Useful macros when performing data audits
 
 # Contents
-- [dbt-audit-helper](#dbt-audit-helper)
-- [Contents](#contents)
 - [Installation instructions](#installation-instructions)
-- [Macros](#macros)
+- [Compare Data Outputs](#compare-data-outputs)
   - [compare\_relations (source)](#compare_relations-source)
   - [compare\_queries (source)](#compare_queries-source)
+  - [compare\_row\_counts (source)](#-compare_row_counts-source)
+- [Compare Columns](#compare-columns)
   - [compare\_column\_values (source)](#compare_column_values-source)
-  - [compare\_relation\_columns (source)](#compare_relation_columns-source)
   - [compare\_all\_columns (source)](#compare_all_columns-source)
   - [compare\_which\_columns\_differ (source)](#compare_which_columns_differ-source)
-  - [compare\_row\_counts (source)](#-compare_row_counts-source)
+  - [compare\_relation\_columns (source)](#compare_relation_columns-source)
+- [Print Output To Logs](#print-output-to-logs)
+- [Use Output For Custom Singular Test](#use-output-for-custom-singular-test)
 
 # Installation instructions
 New to dbt packages? Read more about them [here](https://docs.getdbt.com/docs/building-a-dbt-project/package-management/).
 1. Include this package in your `packages.yml` file — check [here](https://hub.getdbt.com/dbt-labs/audit_helper/latest/) for the latest version number.
 2. Run `dbt deps` to install the package.
 
-# Macros
+# Compare Data Outputs
 ## compare_relations ([source](macros/compare_relations.sql))
```

Comment:
Do you think we should treat compare_relations as a wrapper around compare_queries, and reorder/rephrase them with that in mind? So lean into documentation for compare_queries and then say "Also for convenience compare_relations exists assuming column compatibility etc"

Reply:
That makes sense to me!